Sources/AltSwiftUI/Source/Views/ReadOnly/ProgressView.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  ProgressView.swift
+//  AltSwiftUI
+//
+//  Created by Tsuchiya, Hiroma | Hiroma | TID on 2021/05/27.
+//  Copyright © 2021 Rakuten Travel. All rights reserved.
+//
+
+import UIKit
+
+/// A view that shows the progress towards completion of a task.
+public struct ProgressView: View {
+    public var viewStore = ViewValues()
+    public var body: View {
+        self
+    }
+    
+    var title: String?
+    var label: View?
+    var progress: Float?
+    var total: Float?
+    
+    public init() {}
+    
+    public init(_ title: String) {
+        self.title = title
+    }
+    
+    public init(label: View) {
+        self.label = label
+    }
+    
+    public init(_ progress: Float) {
+        self.progress = progress
+    }
+    
+    public init(value: Float?, total: Float? = 1.0) {
+        self.progress = (value ?? 0) / (total ?? 1.0)
+    }
+    
+    public init(_ title: String, value: Float?, total: Float?) {
+        self.title = title
+        self.progress = (value ?? 0) / (total ?? 1.0)
+    }
+}
+
+extension ProgressView: Renderable {
+    public func createView(context: Context) -> UIView {
+        if (progress != nil) {
+            /// Determinate Progress View
+            if (title != nil) {
+                let titleView = UITextView()
```

Comment:
You can use a UILabel instead of a TextView. A text view can have the contents edited directly by the user (I think this is not the intended behavior right?)

Reply:
You're right. Fixed.